shell/probe/raindrop.ps1

```diff
@@ -40,22 +40,25 @@ while ($true) {
         }
         $Response = Invoke-WebRequest -URI $Api -UseBasicParsing -Headers $Headers -MaximumRedirection 1 -OutFile $Vst -PassThru
         $Test = $Response.BaseResponse.ResponseUri.AbsolutePath.Split("/")[-1].Split("_")[0]
-    
+        $Dat = ""
         if ($Test) {
             if ($CA -eq $Response.BaseResponse.ResponseUri.Authority) {
                 Write-Output "[P] Running $Test [$Vst]"
-                $Code = Execute $Vst   
+                $Code = Execute $Vst
                 $Dat = "${Test}:$Code"
             }
-        } elseif ($Response.BaseResponse.ResponseUri -contains "upgrade") {
+        }
+        Remove-Item $Vst -Force
+        if ($Response.BaseResponse.ResponseUri -contains "upgrade") {
             Write-Output "[P] Upgrade required"
             exit 1
         } else {
-            Remove-Item $Dir -Force -Recurse
             Start-Sleep -Seconds $Sleep
         }
-    } catch { 
+    } catch {
         Write-Error $_
+        Remove-Item $Vst -Force
```

Comment:
You merged a change to that in a previous nocturnal PR so that is different from Nocturnal as of now. we would need to revert that.

So the assumption the probes are making is that the current directory is writeable, not the `.vst` dir only.

I cannot make a guarantee this assumption holds for the installers, which means we need to also go through and re-test all installers with that assumption

Reply:
yep; I see that. let me revert that real quick (should have caught that) to make this consistent and then we can evaluate both.

so the probe's create the .vst directory if not exists, and put the tests inside it during operations. given they are creating it, they should have permission to delete it, right?

Reply:
Linux and Darwin are okay, but the Powershell this will be painful